src/delfipq/models.py

```diff
@@ -1,3 +1,21 @@
-# from django.db import models
+"""
+Models for Delfipq mission.
+"""
+import datetime
+from django.db import models
+from ewilgs.models import Downlink
 
-# Create your models here.
+class Delfipq_L0_telemetry(models.Model): #pylint: disable=C0103
+    """
+    Telemetry Table Delfipq
+    """
+    models.ForeignKey(Downlink, primary_key=True, editable=False, on_delete=models.DO_NOTHING)
+    frame_time = models.IntegerField(default=None, null=True)
+    send_time = models.IntegerField(default=None, null=True)
+    receive_time = models.TimeField(null=False, default = datetime.time )
```

Comment:
Here too.

Reply:
Integer -> Time field for frame_time and send_time